.github/workflows/auto-release.yml

```diff
@@ -39,8 +39,20 @@ jobs:
             git add CHANGELOG.md
             git config --global user.name 'ChiaAutomation'
             git config --global user.email 'automation@chia.net'
-            git commit -m "Updating changelog for $version"
+            git commit -m "chore: Updating changelog for $version"
             git tag $version
             git push origin $version
             git push origin main
           fi
+
+      - name: Auto-update dev dependencies
+        run: |
+          echo "Updating the package.json file now..."
+          npx --yes npm-check-updates --dep dev -u
+          echo "Applying updates to the lock file..."
+          npm update -D
```

Comment:
so my original thought was run `npx --yes npm-check-updates --dep dev` without the -u and grep that into a small thing that checks if there are things that need updating and only run npm update -D if the npx.

maybe this accomplishes the same thing at the end of the day? I guess if theres no updates tehres no change? (thiking out loud)

Reply:
I made an update to address this - take a look